src_python/habitat_sim/simulator.py

```diff
@@ -103,7 +103,7 @@ def _sanitize_config(config: Configuration) -> None:
             )
         )
 
-        config.sim_cfg.requires_textures = any(
+        config.sim_cfg.requires_textures = not config.enable_batch_renderer and any(
```

Comment:
I don't follow here. Shouldn't the user code be setting `sim_cfg.create_renderer==False` if they're not planning to render with this simulator?